cli/pcluster/config/validators.py

```diff
@@ -782,6 +782,27 @@ def s3_bucket_validator(param_key, param_value, pcluster_config):
 
     return errors, warnings
 
+def fsx_lustre_auto_import_validator(param_key, param_value, pcluster_config):
+    errors = []
+    warnings = []
+
+    if param_value != "NONE":
```

Comment:
my understanding is that if a parameter isn't passed in here, `param_value` would be `None` and not `"NONE"`.

Reply:
This check here ensures that if a customer provides an AutoImport Policy of NONE that the s3 cross-region bucket check doesn't occur. Should I also add a "None" check here so the validation doesn't occur too in this case?

Reply:
Yeah, I'd add the check for `None`: `if param_value and param_value != "NONE":`.